x-pack/test/functional/apps/index_management/feature_controls/index_management_security.ts

```diff
@@ -60,7 +60,10 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
       it('should render the "Data" section with index management', async () => {
         await PageObjects.common.navigateToApp('management');
         const sections = await managementMenu.getSections();
-        expect(sections).to.have.length(1);
+
+        // The index_management_user has been given permissions to advanced settings for Stack Management Tests.
+        // https://github.com/elastic/kibana/pull/113078/
```

Comment:
no need of referencing the PR here.

Reply:
I'm confused that `sections` changed from having length 1 to 2, but it looks like the test then only checks the first section.   What's in `sections[1]` ?  Maybe that's the advanced settings?  Maybe covered by another test?